qemu/tests/cfg/balloon_stress.cfg

```diff
@@ -1,26 +1,32 @@
 - balloon_stress:
     type = balloon_stress
     virt_test_type = qemu
-    only Windows
     kill_vm = yes
     repeat_times = 500
     balloon_timeout = 600
-    video_test = win_video_play
```

Comment:
balloon_in_use case use balloon_stress as background test and it still use param "video_test":

- balloon_in_use:
    video_test = win_video_play

Reply:
Modified, Thanks!